### DISCUSSION TITLE
Discussion for building Module ON for Linear Algebra

### DISCUSSION THREAD

**Discussion by jbunn3** [TeamBasedInquiryLearning/library#94] [General]
Hi all,

I have begun updating Module ON by adding to the first section on the dot product. The current two other sections are on projections of vectors and Gram-Schmidt, but they haven't really been built out yet aside from a few definitions. I am proposing making section 2 on Orthogonal Complements, section 3 on Orthogonal Projection, section 4 on Least Squares, and section 5 on Gram-Schmidt. This is in the spirit of what is done in: https://textbooks.math.gatech.edu/ila/index2.html. I like that the big idea is solving Ax\approx b, which is nicely handled once one establishes the benefit of multiplying both side on the left by A^T. I think Gram-Schmidt can be used as a final section for creating nice bases. 

I will add that when I taught Computational Linear Algebra in the Spring, that I taught this content from the GA tech book, mostly through lecture. Most of the student feedback said that they missed TBIL and would have preferred that the course stayed that way :)

I don't plan on pulling anything directly from the GA Tech book, but just using it as a reference. I don't plan on finishing all of this this week, but I want to get a good chunk done, or at least establish a framework. So, I am looking to get some discussion on whether this is a reasonable scope for the TBIL book.

**Comment by fragandi** [comment]
See https://github.com/orgs/TeamBasedInquiryLearning/projects/4

**Comment by jbunn3** [comment]
For reference, the current structure we are suggesting is:

ON1: Dot Products
ON2: Orthogonal Projections
ON3: Orthonormal Bases
Appendix A.4: Least Squares

ON2 would contain orthogonal complements and decomposition, but would only focus on computing projections onto lines.
ON3 will scaffold Gram-Schmidt without making a big deal about the named algorithm. 
We thought least squares would function well as an application section.
